js/google_ima.js

```diff
@@ -524,13 +526,33 @@ require("../html5-common/js/utils/utils.js");
       });
 
       /**
-       * Called by the ad manager controller.  Ad Manager Controller lets the module know that an ad should play now.
-       * @public
+       * debouncing the case when user forwarded ahead and 2 or more ads requests simultaneosly issued
+       * @see https://jira.corp.ooyala.com/browse/PLAYER-3832
```

Comment:
Lets keep tickets out of the jsdocs.  External people can't see our tickets. You can add it as a note above the code in the function.

Reply:
Removed